data/nation/congolesebrazzaville.ruleset

```diff
@@ -37,7 +37,7 @@ init_buildings=""
 init_units=""
 
 conflicts_with = "kongo"
-civilwar_nations = "congolese", "central african"
+civilwar_nations = "congolese", "central_african"
```

Comment:
I don't understand the difference here. Is this GitHub misunderstanding something? Or maybe there's a hidden character?

Reply:
Am I blind? :eyeglasses:

Reply:
"central african" vs "central<underscore>african"

Reply:
I checked the Chadian ruleset and it has it listed as "central african"